SmartCar/Arduino/sketch_main/sketch_main.ino

```diff
@@ -136,6 +177,10 @@ void loop() {
             motor.write(START_POSITION_MOTOR_SERVO);
         }
     }
+
+    //for temperature sensor
+    sensors.requestTemperatures(); // Get temperatures
+    printTemperature(thermometer); //Call 
```

Comment:
Shouldnt be checked and sent each loop since it might block other information that needs to be sent